libs/zflecs/src/zflecs.zig

```diff
@@ -2860,7 +2891,6 @@ test {
 comptime {
     _ = @import("tests.zig");
```

Comment:
What's this doing here? Should we delete it considering that evaluating the definitions in tests.zig is wasted effort. Also, is this causing circular references? And can we drop the branch eval quota in the abi compatibility test if this is removed?

Reply:
This was added by @michal-z 1½ years ago, I guess when initially working on the bindings. I tried removing it and then switching root source file for the test to:
`        .root_source_file = b.path("src/tests.zig"), `

And it seems to work.

And yes, I could now remove the 12325 workaround (I assume that's what you meant) and it still compiles and runs all tests successfully.

@michal-z remember why it was done like this? Any objection to me removing it? :)

Reply:
I don't remember, should be safe to remove.

Reply:
Fixed!